stackdriver/README.md

```diff
@@ -0,0 +1,50 @@
+# Stackdriver Samples
+
+Samples used by Stackdriver documentation. The samples don't work out-of-the-box
```

Comment:
We don't need to point out that *"the samples don't work out-of-the-box"*. Let's remove that sentence.